compiler+runtime/include/cpp/jank/read/parse.hpp

```diff
@@ -12,33 +14,75 @@ namespace jank::runtime
 /* TODO: Rename file to processor. */
 namespace jank::read::parse
 {
-  static option<char> get_char_from_literal(native_persistent_string const &sv)
+  static option<native_persistent_string>
+  parse_character_in_base(native_persistent_string_view const &char_literal, int const base)
+  {
+    try
+    {
+      size_t chars_processed{};
+      auto const codepoint(
+        std::stol(native_persistent_string{ char_literal.data() + 2, char_literal.size() - 2 },
+                  &chars_processed,
+                  base));
+
+      /* Some characters that weren't processed at all.
+       * An example would be when trying to parse `12a` in octal base,
+       * `stol` will only parse `12` and will ignore `a`, returning `2`
+       * in `chars_processed`.
+       * We return none in that case, to represent to the downstream callers
+       * the enability to parse the char literal.
```

Comment:
Please update this comment to convey that we're checking for this case, rather than this being the case. When I read "Some characters that weren't processed at all", I think there's a problem. Perhaps: "We check for characters that were unable to be processed."

Also: `s/enability/inability/`

Overall, I like this helpful comment, though. :+1: